style/infra-style.md

```diff
@@ -0,0 +1,109 @@
+# Infrastructure Team Style Guide Principles
+
+## Why Do We Care
+
+When developers grok each others'* code faster, we can do our work faster.  We read code more often than we write it.
```

Comment:
Is the `*` marking a footnote that I'm not seeing?

Reply:
it's a typo.  I will fix.  thanks.